dali/c_api/c_api.h

```diff
@@ -102,6 +128,15 @@ extern "C" {
    */
   DLL_PUBLIC int64_t* daliShapeAtSample(daliPipelineHandle* pipe_handle, int n, int k);
 
+  /**
+   * @brief Return the type of the 'k' output tensor from tensor list
+   * stored at position `n` in the pipeline.
+   * This function may only be called after
+   * calling Output function.
+   * @remarks Same as calling daliShapeAt(pipe_handle, n)
```

Comment:
what do you mean with `same as calling...` ?

Reply:
That it will give you the same result, because samples have the same type as TensorList

Reply:
Eh. It was supposed to by daliTypeAt. I'm blind.
Janusz wants to leave only one.

Reply:
Just an ask, not a strong blocker if you think that this make API consistent.